CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing to Zed
 
-Thanks for your interest in contributing to Zed, the collaborative platform that is also a code editor!
+Thanks for your interest in contributing to onetothree Zed, the collaborative platform that is also a code editor!
```

Comment:
@ConradIrwin I don't think this was intentional?

Reply:
Nevermind, I got it: https://github.com/zed-industries/zed/commit/575eb792fb2f1c71731a259ec09e7afb58c70528.